File: Sistema/src/app/models/ListaPresenca.Model.ts
export class ListaPresencaModel {
  SesCodi: number;
  UsuCodi: number;
  UsuNome: string;
  UsuNCIM: string;
  LojCodi: number;
  LojNome: string;
  LojNumL: string;
  PreAtiv: boolean;
  PreEmai: boolean;
  MembroLoja: boolean;

  constructor() {
    this.SesCodi = 0;
    this.UsuCodi = 0;
    this.UsuNome = '';
    this.UsuNCIM = '';
    this.LojNome = '';
    this.LojCodi = 0;
    this.LojNumL = '';
    this.PreAtiv = false;
    this.PreEmai = false;
    this.MembroLoja = false;
  }
}

File: API/Models/ListaPresencaModel.cs
namespace API_Visitatus.Models
{
    public class ListaPresencaModel
    {
        public long SesCodi { get; set; }
        public long UsuCodi { get; set; }
        public string? UsuNome { get; set; }
        public string? UsuNCIM { get; set; }
        public long LojCodi { get; set; }
        public string? LojNome { get; set; }
        public string? LojNumL { get; set; }
        public bool PreAtiv { get; set; }
        public bool PreEmai { get; set; }
    }
}
